apps/sms/views/conversation/test/unit/compose_test.js

```diff
@@ -1127,16 +1127,20 @@ suite('compose_test.js', function() {
         assert.isTrue(sendButton.disabled);
       });
 
-      test('enabled if both lock and Compose allow sending', function() {
```

Comment:
__nit:__ I think the old title is still valid, since Compose has it's own check (e.g. size check).